### DISCUSSION TITLE
Opening system settings takes 123 seconds

### DISCUSSION THREAD

**Discussion by ddt3** [rern/rAudio#1229] [General]
I just created a new install of raudio-1 on a 32Gb sd-card following the github readme.

- Added my   wifi info on the sdcard boot partition
- Booted and connected to raudio using my web browser
- Created a Spotify app and added that: Spotify connect working great
But each time I go into Settings -> System the rAudio  icon start flashing for over 2 minutes before showing the actual settings. 

Is this normal? What can I do to avoid this?

**Comment by ddt3** [comment]
no one?

**Comment by rern** [comment]
Was your rAudio connected to the internet? If it's not, online status check could take that long. Next update will fix the issue.
